engines/order_management/spec/lib/order_management/reports/enterprise_fee_summary/parameters_spec.rb

```diff
@@ -0,0 +1,62 @@
+require "spec_helper"
+
+require "date_time_string_validator"
+require "order_management/reports/enterprise_fee_summary/parameters"
+
+describe OrderManagement::Reports::EnterpriseFeeSummary::Parameters do
+  describe "validation" do
+    let(:parameters) { described_class.new }
+
+    it "allows all parameters to be blank" do
+      expect(parameters).to be_valid
+    end
+
+    context "for type of parameters" do
+      it { expect(subject).to validate_date_time_format_of(:start_at) }
```

Comment:
Sorry, I `should` have been more explicit. Pun intended :trollface: 

You have `it { is_expected.to be_empty }` for this. Check it out in https://relishapp.com/rspec/rspec-core/docs/subject/one-liner-syntax

Reply:
I had no idea this method exists. :blush: Thanks! Fixed in https://github.com/openfoodfoundation/openfoodnetwork/pull/3115/commits/2628fe02084e06c40d5a829a222188d36ec6d55d.